src/pages/AboutUs.tsx

```diff
@@ -0,0 +1,105 @@
+import icon from '../assets/icon.svg';
+import contentImg from '../assets/content-balls.svg';
+import headerImg from '../assets/headerImg.svg';
+import lineDivider from '../assets/line-divider.svg';
+import pdfLicense from '../assets/pdf-license.pdf';
+
+const AboutUs = () => {
+  const handleDownload = () => {
+    const link = document.createElement('a');
+    link.href = pdfLicense;
+    link.download = 'licentie-handleiding.pdf';
+    link.click();
+  };
+  return (
+    <article className="aboutus-container">
+      <header className="aboutus-header">
+        <img src={headerImg} alt="header imge" />
+        <h1>over ons</h1>
+        <p className="header-param">
+          Jeu de boules is van origine een echte buitensport in het land van
+          herkomst, Frankrijk. Door het wisselende klimaat in Nederland spelen
+          wij echter van oktober tot april lekker binnen. Wij beschikken over
+          een verwarmde binnen accommodatie!
+        </p>
+        <div className="license-container">
+          <h2>licentie</h2>
+          <p className="license-param">
+            Wil je zelf je digitale licentie downloaden? Bekijk dan hieronder de
+            handleiding:
+          </p>
+          <button onClick={handleDownload}>handleiding licentie</button>
```

Comment:
```suggestion
          <a href={pdfLicense} download="licentie-handleiding.pdf" >handleiding licentie</a>
```
and style it to look like you want